packages/open-api-render-engine/src/renderEngine.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect } from "react";
 import { RenderEngineProps } from "@kyma-project/documentation-component";
 import { OpenApiProps } from "./types";
+import "swagger-ui-dist/swagger-ui.css";
```

Comment:
Remove it.

Reply:
why would I do it, swagger component is literally unusable without default styles

Reply:
But then you must overwrite whole styles if you want add yours.